README.md

```diff
@@ -1,6 +1,9 @@
 blackbird
 =========
 
+[![Circle CI](https://circleci.com/gh/Vagrants/blackbird/tree/master.svg?style=shield)](https://circleci.com/gh/Vagrants/blackbird/tree/master)
+[![License](https://img.shields.io/badge/license-WTFPL-blue.svg)](http://www.wtfpl.net/about/)
+
```

Comment:
Oh.. certainly WTFPL badge is dupuricated !
which badge do you like?
I like blue one.
* ![WTFPL license logo](http://www.wtfpl.net/wp-content/uploads/2012/12/wtfpl-badge-4.png)
* [![License](https://img.shields.io/badge/license-WTFPL-blue.svg)](http://www.wtfpl.net/about/)
* [![License](https://img.shields.io/badge/license-WTFPL-brightgreen.svg)](http://www.wtfpl.net/about/)
* [![License](https://img.shields.io/badge/license-WTFPL-green.svg)](http://www.wtfpl.net/about/)
* [![License](https://img.shields.io/badge/license-WTFPL-yellowgreen.svg)](http://www.wtfpl.net/about/)
* [![License](https://img.shields.io/badge/license-WTFPL-yellow.svg)](http://www.wtfpl.net/about/)
* [![License](https://img.shields.io/badge/license-WTFPL-orange.svg)](http://www.wtfpl.net/about/)
* [![License](https://img.shields.io/badge/license-WTFPL-red.svg)](http://www.wtfpl.net/about/)
* [![License](https://img.shields.io/badge/license-WTFPL-lightgrey.svg)](http://www.wtfpl.net/about/)
* [![License](https://img.shields.io/badge/license-WTFPL-abcdef.svg)](http://www.wtfpl.net/about/)